sample-key-extractor/src/main/java/com/expedia/www/haystack/pipes/kafka/producer/extractor/SampleExtractor.java

```diff
@@ -30,8 +30,8 @@
 
 public class SampleExtractor implements SpanKeyExtractor {
```

Comment:
Log at least traceId in case of an error while extracting

Reply:
make sense. have added the trace id info in logs